python/lsst/sims/GalSimInterface/galSimPSF.py

```diff
@@ -155,7 +156,49 @@ def __init__(self, fwhm=0.6):
         #for r at half the maximum of the PSF
         alpha = fwhm/2.3835
 
-        gaussian1 = galsim.Gaussian(sigma=alpha)
-        gaussian2 = galsim.Gaussian(sigma=2.0*alpha)
+        rms_pixel_size = pixel_scale*pixel_scale/6.0
```

Comment:
The 1/6th came from @RobertLuptonTheGood during our meeting.  I was also wondering about the sqrt(2) factor.  Let's have him confirm just to make sure.